app/views/shared/proposals/_tags_form.html.haml

```diff
@@ -1,13 +1,11 @@
 %fieldset
-  = form_for proposal, url: reviewer_event_proposal_path(event, proposal), html: {role: 'form', remote: true} do |f|
-    .form-group
-      .tag-list
-        = f.label :review_tags
-        - tag_count = event.review_tags.count
-        - if tag_count > 0
+  - if event.review_tags.any?
+    = form_for proposal, url: reviewer_event_proposal_path(event, proposal), html: {role: 'form', remote: true} do |f|
+      .form-group
+        .tag-list
           = f.select :review_tags,
             options_for_select(event.review_tags, proposal.object.review_tags),
             {}, { class: 'multiselect review-tags', multiple: true }
           %button.pull-right.btn.btn-success{:type => "submit"} Update
-        - else
-          None
+  - else
+    None
```

Comment:
This makes me think that there should probably be an event-wide setting for enabling disabling use of tags eventually. We're trying to infer the organizer's intent from whether or not there have been any tags created, but what about when the organizer intends to use tags but there don't happen to be any defined yet?

Until then, I think wrapping this partial in a conditional that toggles the whole thing seems sensible, and so does leaving this logic here that shows 'None' when none are defined. Even if the code never hits this else, if/when we add a setting that explicitly enables/disables tagging, the conditional here will become relevant again.